qemu/tests/x86_cpu_test_dies.py

```diff
@@ -0,0 +1,45 @@
+import random
+
+from avocado.utils import cpu
+
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check guest gets correct multiple vcpu dies
+
+    1) Boot guest with options: -smp n,dies=2x...
+    2) Check cpu dies(only for Linux guest and Intel host)
```

Comment:
`only HostCpuVendor.intel` ?

Reply:
I see you add test result with Win2019 guest,  but comment `only Linux guest` , could you explain more ?

Reply:
Do you mean, for windows guest,  vm can boot with dies successfully is enough ?

Reply:
> `only HostCpuVendor.intel` ?

only check die's topology with HostCpuVendor.intel.

Reply:
> Do you mean, for windows guest, vm can boot with dies successfully is enough ?

Yes, For AMD host or Windows guest, we just need to check booting without error now.

Reply:
> I see you add test result with Win2019 guest, but comment `only Linux guest` , could you explain more ?

just test booting without error.